plaidcloud/utilities/sql_expression.py

```diff
@@ -258,6 +260,7 @@ def get_agg_fn(agg_str):
     if hasattr(sqlalchemy.func, agg_str):
         return getattr(sqlalchemy.func, agg_str)
 
+    # ADT2021 - I believe this line is unreachable, because hasattr(sqlalchemy.func, _) should always be True
```

Comment:
Yep, totally pointless having the check and the `return ident`. It should probably just read `return getattr(sqlalchemy.func, add_str)` and omit the always True check